src/tss2-tcti/tctildr-dl.c

```diff
@@ -112,7 +112,7 @@ handle_from_name(const char *file,
         if (*handle != NULL) {
             return TSS2_RC_SUCCESS;
         } else {
-            LOG_DEBUG("Could not load TCTI file \"%s\": %s", file, dlerror());
```

Comment:
We don't want to change this to an error. Firstly, it is not fatal. Secondly, it is expected that different tcti name variants are not found while tctildr tries different variants (`<child_name>`, `libtss2-tcti-<child_name>.so.0`, ...)

Reply:
whoops, that was unattended, I thought I cleaned up all my cruft